### ISSUE TITLE
Kill a machine

### ISSUE THREAD

**Issue by JeromeVi** [vweevers/node-docker-machine#6]
I can't find how to do "docker-machine kill myMachine" is it a way to do it with node-docker-machine ?

**Comment by vweevers** [comment]
The `kill` command is not implemented. A PR is welcome.

**Comment by JeromeVi** [comment]
Ok I'll check if I can implement that , any help is welcome

**Comment by JeromeVi** [comment]
Ok so I think this code should do the work because when we do `docker-machine kill vbox-vial` the state of vbox-vial is `stopped`
<img width="542" alt="capture d ecran 2017-01-18 a 10 28 30" src="https://cloud.githubusercontent.com/assets/22610050/22058361/e6cce768-dd68-11e6-921f-91d8d78628b1.png">


```
  static kill(name, done) {
    Machine.command(['kill', name], (err) => {
      if (HOST_NON_EXISTENT.test(err)) {
        done(new Error(`Docker host "${name}" does not exist`))
      } else if (ALREADY_STOPPED.test(err)) {
        done()
      } else {
        done(err)
      }
    })
  }    
```

**Comment by vweevers** [comment]
Is the `ALREADY_STOPPED` error the same? When you try to kill a machine that isn't running, what message do you get?

**Comment by JeromeVi** [comment]
I got that `Machine "vbox-vial" is already stopped.`

**Comment by vweevers** [comment]
Great, go ahead and open a PR :)

**Comment by JeromeVi** [comment]
I don't know how to do it right now, I'll try later ;)

**Comment by JeromeVi** [comment]
Yes I really don't know how to do a pull request

**Comment by vweevers** [comment]
No worries, there are plenty tutorials online to help you get started. This one is pretty extensive and easy to follow: https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github

**Comment by JeromeVi** [comment]
I guess I have done the PR :)

**Comment by vweevers** [comment]
Released in 2.1.0.
